wicket-core/src/main/java/org/apache/wicket/page/DefaultPageLockManager.java

```diff
@@ -222,3 +227,12 @@ Supplier<ConcurrentMap<Integer, PageAccessSynchronizer.PageLock>> getLocks()
 		return locks;
 	}
 }
+
+static class PageLockedException extends Exception
+{
```

Comment:
```suggestion
class PageLockedException extends Exception
{
```
@martin-g this is outside the surrounding class, so it is already static. The compiler also complains:

```
Error:  /home/runner/work/wicket/wicket/wicket-core/src/main/java/org/apache/wicket/page/DefaultPageLockManager.java:[231,8] modifier static not allowed here
```

Reply:
> this is outside the surrounding class

who does this ?! :-)
why not create its own class ? because you will need to add a license at the top ? :-)

Reply:
🖐️ 

The exception has no reuse outside the DefaultPageLockManager, I thought about making it a local class, but then you get those enclosures in your stack traces, which I don't like. So this should get both requirements: no strange stack trace names and no reuse.